docs/tutorials/e2e/boost/provideData.md

```diff
@@ -33,41 +40,243 @@ curl --location 'http://localhost/bob/management/v2/assets' \
 }' | jq
 ```
 
+Just to be sure, that the asset was created succesfully, Bob can check the asset using the following `curl` command:
+
+```shell
+curl -X POST http://dataprovider-controlplane.tx.test/management/v2/assets/request -H "x-api-key: TEST2" -H "content-type: application/json" | jq
+```
+
+The result shows the already existing assets and the newly created asset.
+
+```json
+[
+  {
+    "@id": "registry-asset",
+    "@type": "edc:Asset",
+    "edc:properties": {
+      "edc:type": "data.core.digitalTwinRegistry",
```

Comment:
This is messy - not sure where these assets were supposedly created but the way that's done is not how it's standardized...